tls-tutorial/doc/tls_in_udfs.md

```diff
@@ -0,0 +1,225 @@
+# TLS in User Defined Functions (UDFs)
+
+## Before you Start
+
+If you haven't already, I recommend you read our ["TLS introduction"](tls_in_udfs.md) to familiarize yourself with the basic concepts of TLS.
+
+You should familiarize yourself with Exasol's [BucketFS](https://docs.exasol.com/db/latest/database_concepts/bucketfs/bucketfs.htm) filesystem.
+
+You will also need:
+
+1. A Java Development Kit (JDK 11 or later, we recommend [OpenJDK](https://openjdk.org/install/))
```

Comment:
Won't a JRE be sufficient?

Reply:
For running - yes. But the JDK has the added benefits of JavaDoc and the chance to look into the SDK code to see what's going on. After all, this is tutorial mainly targeted at developers.